### DISCUSSION TITLE
Are there types for `@preact/prerender-data-provider`?

### DISCUSSION THREAD

**Discussion by orangecoloured** [preactjs/preact-cli#1801] [Q&A] [answered]
I'm using TypeScript for my project and after adding the `@preact/prerender-data-provider` I found out that it's not typed.
And it seems that there are no types available. Did I miss it?

The main issue is that `build` fails with no types.

**Comment by rschristian** [comment, marked as answer]
No, it does not provide any TS types.

Contributions welcome, if you wanted to add them.

> The main issue is that build fails with no types.

I think that should only be the case if you've set `"strict": true` in your `tsconfig.json`, otherwise imports w/out types should be `any` AFAIK.
